nfqueue_lt_1.12.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/twistlock/go-nfqueue/internal/unix"
 
-	"github.com/mdlayher/netlink"
+	"github.com/twistlock/netlink"
```

Comment:
for consistency can you also fix line 46 to do the same?

Reply:
I'm not sure I understand... line 46 is `nfqueue.Con = con`